src/lib/core/components/computations/getAppDisplayProperties.tsx

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+import { useStudyMetadata } from '../..';
+import { useCollectionVariables } from '../../hooks/study';
+import { Computation } from '../../types/visualization';
+
+const ABUNDANCE_METHODS = ['median', 'q3', 'variance', 'max'];
+const ALPHA_DIV_METHODS = ['shannon', 'simpson', 'evenness'];
+
+export function useAppPropertiesForDisplay(
```

Comment:
What do you think about having each plugin define it's own useAppPropertiesForDisplay or similar function in the exported plugin? The biggest obvious con to me is that would introduce duplicated code. But the big advantage i see is that as these configurations continue to diverge, we can keep any app-specific logic or verbiage within the plugin file.

Reply:
I didn't wire up the methods in this hook yet because I didn't see an obvious way to do so. But handling the display/description logic within the plugins themselves could make sense.

Reply:
For instance, in [AlphaDivConfiguration](https://github.com/VEuPathDB/web-eda/blob/organize-vizs-1095/src/lib/core/components/computations/plugins/alphaDiv.tsx), we currently have an input box to allow the user to "name" their app instance. Instead, we can generate on based on the config.